frontend/src/components/users/Users.tsx

```diff
@@ -6,44 +6,46 @@ import Search from "components/search/Search";
 export type IUsers = IUser[];
 
 const Users = () => {
-	const { handleChange, filteredUsers, users } = useUsers();
+    const { handleChange, filteredUsers, users } = useUsers();
 
-	const UsersList = () => {
-		return (
-			<>
-				{users.length === 0 ? (
-					<span style={{ color: "white" }}>Loading...</span>
-				) : (
-					<Users />
-				)}
-			</>
-		);
-	};
+    const Users = () => (
+        <>
+            {filteredUsers.map(({ id, username }: IUser) => (
```

Comment:
Don't destructure, just pass the whole user

```js
filteredUsers.map((user: IUser) => (
  <User key={user.id} {...user}/>
))
```

Reply:
That's not my solution, but i think about it, i create it inside, but i need React fragment, for that map.

I try to do it better.

Reply:
Ye, i forget about something with that, and now it's looks better